### ISSUE TITLE
Include a UI test with keyboard selection of elements

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#1152]
Following on from #1151, I think we need to incorporate (probably into the Ray Charles test, because it is currently less "crowded") a test that it is possible to select and use various UI elements (buttons) using the keyboard only. For example, something like using the tab key to cycle through UI elements and be sure that they are at least focused, if not actionable.

**Comment by Jaifroid** [comment]
I'd probably suggest testing the library button, sending the Enter key. I don't think it would be possible to test the file/folder selectors (because you can't actually "press" them, it's not possible to programme the native file selection dialogues, as they happen in the OS, not in Selenium).

**Comment by Jaifroid** [comment]
@Rishabhg71 I've assigned you to this issue, but it's not at all an urgent issue. Just a "would be desirable". More important to work on the libzim integration and not let this one distract you, IMHO.

**Comment by skushagra9** [comment]
@Jaifroid, is there any critical issue which needs to be solved ??

**Comment by Rishabhg71** [comment]
@skushagra9 Take this issue if you want. If you solve this issue I think we can include it in v4.0

**Comment by Jaifroid** [comment]
Agreed. @skushagra9, do you have any experience of testing with Selenium, or are you willing to read up about it? It's probably not a very easy first issue, but not hugely difficult either. The trickiest thing with testing scripts is that timings and timeouts in an asynchronous context are not always obvious.

**Comment by skushagra9** [comment]
Ok, I will definately give it a try

**Comment by Jaifroid** [comment]
Thanks @skushagra9. Please check [CONTRIBUTING.md](https://github.com/kiwix/kiwix-js/blob/main/CONTRIBUTING.md) for how to set up the Repo. I suggest you only add this test to https://github.com/kiwix/kiwix-js/blob/main/tests/e2e/spec/legacy-ray_charles.e2e.spec.js, as we don't need it to run in both ZIMs. I suggest you add the test outlined in https://github.com/kiwix/kiwix-js/issues/1152#issuecomment-1807161623. As you are a new contributor, you will have to notify one of us to get the tests to run when you add commits to your PR. They will run on GitHub, but won't run on Browser Stack initially, but the GitHub test will determine whether your code is working.

**Comment by skushagra9** [comment]
Ok Thanks
